ui.config/src/main/content/jcr_root/apps/saas-aem-module/osgiconfig/config/org.apache.sling.jcr.repoinit.RepositoryInitializer~saas-content-reader.cfg.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": [
+    "create path (rep:AuthorizableFolder) /home/users/system/saas",
```

Comment:
do we need 2 configs? https://github.com/valtech-ch/saas-aem-module/pull/20/files#diff-416bdc84e66ce6bb60dbe878d1759e52a6f06786db0506b3b4193677cb12edec as well?

Reply:
no, it is not. i'll remove the .config file